client/components/jetpack/jetpack-product-info/recommendation-tags.tsx

```diff
@@ -0,0 +1,29 @@
+import { JetpackTag } from '@automattic/calypso-products';
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import { Tags } from 'calypso/my-sites/plans/jetpack-plans/product-lightbox/icons/tags';
+
+type Props = { tags: JetpackTag[] };
+
+export const JetpackProductInfoRecommendationTags: FunctionComponent< Props > = ( {
+	tags: tags,
+} ) => {
+	const translate = useTranslate();
+
+	return (
+		<div className="jetpack-product-info__tags">
+			<span className="jetpack-product-info__tags-label">{ translate( 'Great for:' ) }</span>
+
+			<ul className="jetpack-product-info__tags-list">
+				{ tags.map( ( tag ) => (
```

Comment:
How about we destructure the tag?

```
{ tags.map( ( { tag, label } ) => (
	<li className="jetpack-product-info__tags-list-item" key={ tag }>
		<span aria-hidden="true">{ Tags[ tag ] }</span>
		<p>{ label }</p>
	</li>
) ) }
```

Reply:
Nice suggestion. this has been applied 👍